Template is broken with latest `libcosmic`

The build errors out:
```
snowball-file/crates/snowvault-gui on  master [!?⇕] is 📦 v0.1.0 via 🦀 v1.82.0 
❯ just
cargo build --release 
   Compiling snowvault_gui v0.1.0 (/home/skyfall/snowball-file/crates/snowvault_gui)
error[E0053]: method `context_drawer` has an incompatible type for trait
   --> crates/snowvault_gui/src/app.rs:130:33
    |
130 |     fn context_drawer(&self) -> Option<Element<Self::Message>> {
    |                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `ContextDrawer<'_, Message>`, found `Element<'_, Message, Theme, ...>`
    |
    = note: expected signature `fn(&AppModel) -> Option<cosmic::app::context_drawer::ContextDrawer<'_, app::Message>>`
               found signature `fn(&AppModel) -> Option<cosmic::iced_core::Element<'_, app::Message, cosmic::Theme, cosmic::iced_renderer::fallback::Renderer<cosmic::iced_wgpu::Renderer, iced_tiny_skia::Renderer>>>`
help: change the output type to match the trait
    |
130 |     fn context_drawer(&self) -> Option<cosmic::app::context_drawer::ContextDrawer<'_, app::Message>> {
    |                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

error[E0599]: no method named `set_context_title` found for mutable reference `&mut AppModel` in the current scope
   --> crates/snowvault_gui/src/app.rs:210:22
    |
210 |                 self.set_context_title(context_page.title());
    |                      ^^^^^^^^^^^^^^^^^ method not found in `&mut AppModel`

Some errors have detailed explanations: E0053, E0599.
For more information about an error, try `rustc --explain E0053`.
error: could not compile `snowvault_gui` (bin "snowvault_gui") due to 2 previous errors
error: Recipe `build-debug` failed on line 38 with exit code 101
```
Using `libcosmic` commit `a355a049d917823dafcd6669cad6afa6983bc6c6`.

--- a/src/app.rs
+++ b/src/app.rs
@@ -2,7 +2,7 @@
 
 use crate::config::Config;
 use crate::fl;
-use cosmic::app::{Core, Task};
+use cosmic::app::{context_drawer, Core, Task};
 use cosmic::cosmic_config::{self, CosmicConfigEntry};
 use cosmic::iced::alignment::{Horizontal, Vertical};
 use cosmic::iced::{Alignment, Length, Subscription};
@@ -114,7 +114,7 @@ impl Application for AppModel {
             menu::root(fl!("view")),
             menu::items(
                 &self.key_binds,
-                vec![menu::Item::Button(fl!("about"), MenuAction::About)],
+                vec![menu::Item::Button(fl!("about"), None, MenuAction::About)],
             ),
         )]);
 
@@ -127,13 +127,17 @@ impl Application for AppModel {
     }
 
     /// Display a context drawer if the context page is requested.
-    fn context_drawer(&self) -> Option<Element<Self::Message>> {
+    fn context_drawer(&self) -> Option<context_drawer::ContextDrawer<Self::Message>> {
         if !self.core.window.show_context {
             return None;
         }
 
         Some(match self.context_page {
-            ContextPage::About => self.about(),
+            ContextPage::About => context_drawer::context_drawer(
+                self.about(),
+                Message::ToggleContextPage(ContextPage::About),
+            )
+            .title(fl!("about")),
         })
     }
 
@@ -205,9 +209,6 @@ impl Application for AppModel {
                     self.context_page = context_page;
                     self.core.window.show_context = true;
                 }
-
-                // Set the title of the context drawer.
-                self.set_context_title(context_page.title());
             }
 
             Message::UpdateConfig(config) => {
@@ -277,14 +278,6 @@ pub enum Page {
 pub enum ContextPage {
     #[default]
     About,
-}
-
-impl ContextPage {
-    fn title(&self) -> String {
-        match self {
-            Self::About => fl!("about"),
-        }
-    }
 }
 
 #[derive(Clone, Copy, Debug, Eq, PartialEq)]

--- a/src/main.rs
+++ b/src/main.rs
@@ -12,7 +12,11 @@ fn main() -> cosmic::iced::Result {
     i18n::init(&requested_languages);
 
     // Settings for configuring the application window and iced runtime.
-    let settings = cosmic::app::Settings::default();
+    let settings = cosmic::app::Settings::default().size_limits(
+        cosmic::iced::Limits::NONE
+            .min_width(360.0)
+            .min_height(180.0),
+    );
 
     // Starts the application's event loop with `()` as the application's flags.
     cosmic::app::run::<app::AppModel>(settings, ())

